yarn.lock

```diff
@@ -1313,28 +1313,28 @@
   dependencies:
     "@date-io/core" "^2.13.1"
 
-"@date-io/date-fns@^2.13.1":
+"@date-io/date-fns@^2.11.0", "@date-io/date-fns@^2.13.1":
```

Comment:
Does this come from `@mui/x-date-pickers`? If yes, should it use `2.13.1` instead?

Reply:
Good point, I forgot to remove the `@date-io` deps from the lab

Reply:
And the update from `2.11.0` to `2.13.1` on X will come on Monday with Renovate :+1: